Sources/ForageSDK/Component/ForagePINTextField/ForagePINTextField.swift

```diff
@@ -96,20 +96,34 @@ public class ForagePINTextField: UIView, Identifiable {
         return sv
     }()
     
-    private lazy var textField: VGSTextField = {
-        let tf = VGSTextField()
-        tf.translatesAutoresizingMaskIntoConstraints = false
-        tf.textColor = UIColor.black
-        tf.font = UIFont.systemFont(ofSize: 14, weight: .regular)
-        tf.autocorrectionType = .no
-        tf.borderWidth = 0.25
-        tf.borderColor = UIColor.lightGray
-        tf.padding = UIEdgeInsets(top: 0, left: 8, bottom: 0, right: 0)
-        tf.accessibilityIdentifier = "tf_forage_pin_text_field"
-        tf.isAccessibilityElement = true
-        return tf
+    private lazy var textField: PINVaultTextField = {
+        let vaultType = LDManager.shared.getVaultType()
+        
+        var tf: PINVaultTextField?
+        
+        if (vaultType == VaultType.vgsVaultType) {
+            tf = VGSTextFieldWrapper()
+        } else if (vaultType == VaultType.btVaultType) {
+            tf = BasisTheoryTextFieldWrapper()
+        } else {
+            fatalError("Unexpected vaultType: \(vaultType)")
```

Comment:
Unlikely it ever happens, but I defaulted to VGS in this scenario. I may add some logging here as well just in case when I implement datadog.